arangod/RocksDBEngine/RocksDBTransactionManager.h

```diff
@@ -67,8 +69,17 @@ class RocksDBTransactionManager final : public TransactionManager {
 
   uint64_t getActiveTransactionCount() override { return _nrRunning; }
 
+  // temporarily block all new transactions
+  virtual bool holdTransactions(uint64_t timeout) {return _rwLock.writeLock(timeout);};
```

Comment:
`virtual` not required here, but should be marked as `override` instead.
```suggestion
  bool holdTransactions(uint64_t timeout) override {return _rwLock.writeLock(timeout);}
```